src/components/chips/demoBasicUsage/style.scss

```diff
@@ -40,10 +40,9 @@
         }
       }
     }
-    &:not(.md-readonly) {
-      md-chip-template {
-        padding-right: 5px;
-      }
+
```

Comment:
I don't think you can *always* add this; I think it should have the padding only if `md-removable="true"`. Right?